src/services/api/hooks.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'lib/gqty';
+
+import { useSelectedCircle } from 'recoilState';
+
+export const useCurrentOrg = () => {
```

Comment:
It's wierd to me to have hooks in this folder, it's two levels deep - lets have it be in `hooks/` or have another top level for `gqty`